kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -170,6 +170,7 @@
       const { createSnackbar } = useSnackbar();
 
       const router = getCurrentInstance().proxy.$router;
+      const { $route } = getCurrentInstance().proxy;
```

Comment:
This gets me wondering if maybe we ought to use the `$rout(e|er)` variables when we pull them out. It feels like we've been removing the `$` when pulling the route/store/router out of `getCurrentInstance`.

Not a blockin

Reply:
Yes, I was also wondering that, I also prefer using with the `$` as it feels a little more similar to what we do in the options api :sweat_smile:. I just updated the router variable to use the `$` :).